.github/ISSUE_TEMPLATE/stale.yml

```diff
@@ -0,0 +1,33 @@
+---
```

Comment:
Should this file be `.github/stale.yml` instead? I think the `ISSUE_TEMPLATE` directory is just for github issue templates.

Reply:
Don't know how I created the file in the wrong directory and didn't noticed 🤦‍♂ 
Thanks 👍